openapi/servicenow/client.bal

```diff
@@ -0,0 +1,477 @@
+    # + accept - Data format of the response body.
+    # + return - If successful, returns a JSON object depends on specified table and specified request parameters. Otherwise returns the relevant error.
+    remote isolated function aggregate(string tableName, string? nameValuePairs = (), string? sysparmAvgFields = (), string? sysparmMinFields = (), string? sysparmMaxFields = (), string? sysparmSumFields = (), string? sysparmCount = (), string? sysparmDisplayValue = (), string? sysparmGroupBy = (), string? sysparmHaving = (), string? sysparmOrderby = (), string? sysparmQuery = (), string? accept = ()) returns json|error {
+        string  path = string `/api/now/stats/${tableName}`;
+        map<anydata> queryParam = {"name-value pairs": nameValuePairs, "sysparm_avg_fields": sysparmAvgFields, "sysparm_min_fields": sysparmMinFields, "sysparm_max_fields": sysparmMaxFields, "sysparm_sum_fields": sysparmSumFields, "sysparm_count": sysparmCount, "sysparm_display_value": sysparmDisplayValue, "sysparm_group_by": sysparmGroupBy, "sysparm_having": sysparmHaving, "sysparm_orderby": sysparmOrderby, "sysparm_query": sysparmQuery};
+        path = path + check getPathForQueryParam(queryParam);
+        map<any> headerValues = {"Accept": accept};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        json response = check self.clientEp-> get(path, accHeaders, targetType = json);
+        return response;
+    }
+    # Retrieves the specified import staging record and resulting transformation result.
+    #
+    # + stagingTableName - Name of the table from which to obtain the import data.
+    # + sys_id - Sys_id of the record that contains the data.
+    # + accept - Data format of the response body.
+    # + return - If successful, returns a JSON object. Otherwise returns the relevant error.
+    remote isolated function getImportSetTable(string stagingTableName, string sys_id, string? accept = ()) returns json|error {
+        string  path = string `/api/now/import/${stagingTableName}/${sys_id}`;
+        map<any> headerValues = {"Accept": accept};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        json response = check self.clientEp-> get(path, accHeaders, targetType = json);
+        return response;
+    }
+    # Inserts incoming data into a specified staging table and triggers transformation based on predefined transform maps in the import set table.
+    #
+    # + stagingTableName - Name of the table from which to import the data.
+    # + payload - Name-value pairs to insert in the import fields in JSON format.
+    # + accept - Data format of the response body.
+    # + contentType - Data format of the request body.
+    # + return - If successful, returns a JSON object. Otherwise returns the relevant error.
+    remote isolated function insertImportSetTable(string stagingTableName, json payload, string? accept = (), string? contentType = ()) returns json|error {
```

Comment:
The function name must be `createImportSet`

Reply:
Fixed by https://github.com/kasthuriraajan/ballerinax-openapi-connectors/commit/7d722e1656dd4084768a6a6f0aab1753b97a5d98